guides/doc-Installing_Server_on_Red_Hat/master.adoc

```diff
@@ -54,9 +54,12 @@ include::common/assembly_using-external-databases.adoc[leveloffset=+2]
 
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 
-
 include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
 
+ifdef::katello,satellite[]
```

Comment:
@ekohl I'm wondering that the procedure would fit better into the Puppet Guide we have in the downstream. Any downsides I should observe?

Reply:
Consulted with Ondra Ezr, I will include it in Admin Guide.